packages/kit/types/index.d.ts

```diff
@@ -1110,6 +1111,8 @@ export interface HttpError {
 
 /**
  * Create a `Redirect` object. If thrown during request handling, SvelteKit will return a redirect response.
+ * Make sure you're not catching the thrown redirect, which results in a noop.
+ *
```

Comment:
We mostly don't have a blank line before the `@param` tags and you're not adding one above, so probably better to leave it out for consistency
```suggestion
```